tests/vmi_lifecycle_test.go

```diff
@@ -372,6 +377,11 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			Context("without k8s secret", func() {
 				It("[test_id:1629][posneg:negative]should not be able to start virt-launcher pod", func() {
 					vmi = libvmifact.NewCirros()
+					arch := testsuite.TranslateBuildArch()
+					// Use Alpine instead of cirros as cirros is not supported on s390x.
+					if checks.IsS390X(arch) {
```

Comment:
Since you are repeating this pattern multiple times, it might beneficial to create a function that constructs a tiny guest based on the arch

Reply:
done